src/test_log/live_video_feed_object_recognition.cpp

```diff
@@ -0,0 +1,129 @@
+/**
```

Comment:
Tip: I would like to see some more info about these tests. What do they test, and how? The strings used as test case items are OK, but descriptions will really improve readablity.
As (as far as I know) the wiki does (sadly) not make any notices about the documentation standard for tests, you should consider this as a improvement point.